sdk/src/feature_set.rs

```diff
@@ -143,7 +143,7 @@ pub mod zk_token_sdk_enabled {
 }
 
 pub mod curve25519_syscall_enabled {
-    solana_sdk::declare_id!("7rcw5UtqgDTBBv2EcynNfYckgdAaH1MAsCjKgXMkN7Ri");
+    solana_sdk::declare_id!("eca6zf6JJRjQsYYPkBHF3N32MTzur4n2WL4QiiacPCL");
```

Comment:
we can't rekey like this since it's already activated on testnet and devnet. upgrading can kill those clusters. we need to add a new gate that activates the fix, activate it on testnet and devnet, then activate both the global feature gate and fix feature gate simultaneously on mb

Reply:
Ah okay, makes sense. Will update it!